BaikalDataGenerator/Source/render.h

```diff
@@ -22,27 +22,36 @@ THE SOFTWARE.
 
```

Comment:
lets rename file and class to data_generator_impl and DataGeneratorImpl correspondingly